package.json

```diff
@@ -21,18 +21,26 @@
   "bugs": {
     "url": "https://github.com/codealchemist/youtube-audio-server/issues"
   },
+  "files": ["!/temp-audio/**", "src/*", "!/src/test"],
   "homepage": "https://github.com/codealchemist/youtube-audio-server#readme",
   "dependencies": {
     "express": "^4.17.1",
     "express-no-favicons": "0.0.1",
+    "ffmetadata": "^1.6.0",
     "fluent-ffmpeg": "^2.1.2",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
+    "qs": "^6.9.1",
+    "retriable-download": "^2.1.0",
```

Comment:
Can we look into using something more widely used?
Like:
https://www.npmjs.com/package/download

Reply:
still need retriable-download for cover art

Reply:
We can do something like this:
`download(imgUrl, { retry: 3 })`

https://www.npmjs.com/package/download
It has 3.5M weekly downloads and a clear API.
Weights only 6KB.

https://www.npmjs.com/package/retriable-download
It has 48 weekly downloads and the same weight.
We should be safer with the first one for the same cost in KB 😄

Reply:
BTW, that returns a stream we can then pipe to a `createWriteStream`.

Reply:
nah, i just want to keep the source short, it's a simple feature btw

Reply:
the purpose is just save the image to temp folder and run *ffmpeg*, `retriable-download` handled it in a single line of code